### ISSUE TITLE
Cover art not working on Windows

### ISSUE THREAD

**Issue by herbmillerjr** [herbmillerjr/Celeste#55]
Cover art shows for the `!song` command on Linux, but not on Window. The image data is present in the tags in the files, but for some reason Qt can't read them.

**Comment by herbmillerjr** [comment]
```285     if (SUCCEEDED(MFGetService(mediaSource, MF_PROPERTY_HANDLER_SERVICE, IID_PPV_ARGS(&m_content)))) {
286         DWORD cProps;
287         if (SUCCEEDED(m_content->GetCount(&cProps))) {
288             for (DWORD i = 0; i < cProps; i++)
289             {
290                 PROPERTYKEY key;
291                 if (FAILED(m_content->GetAt(i, &key)))
292                     continue;
293                 bool common = true;
294                 if (key == PKEY_Author) {
295                     m_availableMetaDatas.push_back(QMediaMetaData::Author);
296                 } else if (key == PKEY_Title) {
297                     m_availableMetaDatas.push_back(QMediaMetaData::Title);
298                 } else if (key == PKEY_Media_SubTitle) {
299                     m_availableMetaDatas.push_back(QMediaMetaData::SubTitle);
300                 } else if (key == PKEY_ParentalRating) {
301                     m_availableMetaDatas.push_back(QMediaMetaData::ParentalRating);
302                 } else if (key == PKEY_Media_EncodingSettings) {
303                     m_availableMetaDatas.push_back(QMediaMetaData::Description);
304                 }```

Well, unfortunately my guess was correct. When using Windows Media Foundation, there is no PKEY_Cover_Art_Image, or anything similar. So Celeste being unable to display cover art is a Windows limitation. Guess I'll have to roll my own. 

`97         metadataKeys->insert(GST_TAG_IMAGE, QMediaMetaData::CoverArtImage);`

Linux, on the other hand, relies on Gstreamer, which can pull the image. Gstreamer > WMF, apparently.

**Comment by herbmillerjr** [comment]
Fixed by f1131300d415320428b6b90b84264ec3b820a808
